docs/supported_apis/dataframe_supported.rst

```diff
@@ -6,671 +6,40 @@ of an operation that is listed as not implemented, feel free to open an issue on
 `GitHub repository`_, or give a thumbs up to already created issues. Contributions are
 also welcome!
 
-The following table is structured as follows: The first column contains the method name.
-The second column contains link to a description of corresponding pandas method.
-The third column is a flag for whether or not there is an implementation in Modin for
-the method in the left column. ``Y`` stands for yes, ``N`` stands for no, ``P`` stands
-for partial (meaning some parameters may not be supported yet), and ``D`` stands for
-default to pandas.
+The following table is structured as follows: The first column contains the method name,
+the second column - the parameter name of this method, and other columns contain
+different flags describing particular properties of method parameters for a concrete
+execution.
 
-.. note::
-    Currently third column reflects implementation status for Ray and Dask engines. By default, support for a method
-    in the Omnisci engine could be treated as ``D`` unless ``Notes`` column contains additional information. Similarly,
-    by default ``Notes`` contains information about ``Ray`` and ``Dask`` engines unless ``Omnisci`` is explicitly mentioned.
+The flags stand for the following:
 
-+----------------------------+---------------------------+------------------------+----------------------------------------------------+
```

Comment:
So we just remove all spreadsheets and leave i/o functions only which are in new format - maybe we can stick to mix mode and have fresh-formatted spreadsheet for i/o and old-fashioned spreadsheets for others?

Reply:
This change came from #4286 and was added for demonstration purposes only and removed now (this PR covers IO functions docstrings only).

Reply:
Ah, got it, thanks. I think we can then proceed with what can be called mix mode, i.e. IO functions will be in the new format, for other pages I will file corresponding issues to start converting from rst form

Reply:
Actually yes, once this PR is merged, #4286 can be reworked to operate with IO supported APIs tales only for now.